experimental/packages/sdk-logs/src/LogRecord.ts

```diff
@@ -73,15 +75,26 @@ export class LogRecord implements ReadableLogRecord {
 
   constructor(logger: Logger, logRecord: logsAPI.LogRecord) {
     const {
-      timestamp = Date.now(),
+      timestamp,
+      observedTimestamp,
       severityNumber,
       severityText,
       body,
       attributes = {},
       context,
     } = logRecord;
 
-    this.hrTime = timeInputToHrTime(timestamp);
+    if (timestamp && observedTimestamp) {
+      this.hrTime = timeInputToHrTime(timestamp);
+      this.hrTimeObserved = timeInputToHrTime(observedTimestamp);
+    } else {
+      const now = hrTime();
```

Comment:
hrTime will be drifted in some situations. https://github.com/open-telemetry/opentelemetry-js/issues/3355 for details